docs/en/CSMM/advanced-feature-guide-fallingblocks.md

```diff
@@ -72,37 +74,44 @@ Our resulting command for the hook is:
 {{/each}}
 ```
 
-Be aware that this command should completely replace our temporary `wait(1)` that we entered to begin with.
+NOTE: Be aware that this command should completely replace our temporary `wait(1)` that we entered to begin with.
```

Comment:
::: warning
Be aware that this command should completely replace our temporary `wait(1)` that we entered to begin with.
:::